packages/ui/src/components/va-menu-list/VaMenuList.stories.ts

```diff
@@ -0,0 +1,162 @@
+import { VaDivider, VaButton, VaDropdown, VaIcon, VaAvatar } from '../../components'
+import VaMenuList from './VaMenuList.vue'
+import VaMenuItem from './components/VaMenuItem.vue'
+import VaMenuGroup from './components/VaMenuGroup.vue'
+import { addText } from '../../../.storybook/interaction-utils/addText'
+
+export default {
+  title: 'VaMenuList',
+  component: VaMenuList,
+  tags: ['autodocs'],
+}
+
+export const Default = () => ({
+  components: { VaMenuList },
+  data: () => ({
+    options: ['Option 1', 'Option 2', 'Option 3'],
+  }),
+  template: '<VaMenuList :options="options" />',
+})
+
+export const IconSlot = () => ({
+  components: { VaMenuList, VaAvatar },
+  data: () => ({
+    options: ['Option 1', 'Option 2', 'Option 3'],
+  }),
+  template: `
+  <VaMenuList :options="options">
+    <template #left-icon>
+      [left-icon]
+    </template>
+    <template #icon>
+      [icon]
+    </template>
+  </VaMenuList>
+  `,
+})
+addText(
+  IconSlot,
+  'There is no slot for right icon, only for left one.',
+  'broken',
+)
+
+const longGroupName = 'This is a very long group name that should be truncated'.repeat(10)
+export const Groups = () => ({
+  components: { VaMenuList, VaAvatar },
+  data: () => ({
+    options: [
+      { id: '0', text: 'one', value: 'one', icon: 'accessible_forward', rightIcon: '', group: '' },
+      { id: '1', text: 'two', value: 'two', icon: '', rightIcon: 'home', group: longGroupName },
+      { id: '2', text: 'three', value: 'three', icon: 'accessible_forward', rightIcon: '', group: longGroupName },
+      { id: '3', text: 'four', value: 'four', icon: '', rightIcon: 'accessible_forward', group: 'A2' },
+      { id: '4', text: 'five', value: 'five', icon: 'search', rightIcon: '', group: 'A2' },
+    ],
+    value: false,
+  }),
+  template: `
+  <VaMenuList :options="options">
+  </VaMenuList>
+  `,
+})
+addText(
```

Comment:
I added my notes to story descriptions. We probably need to fix these before we merge.